staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -234,6 +234,21 @@ type Store struct {
 	// If set, DestroyFunc has to be implemented in thread-safe way and
 	// be prepared for being called more than once.
 	DestroyFunc func()
+
+	// CorruptObjDeleter implements unsafe deletion flow to enable deletion
+	// of corrupt object(s), it makes an attempt to perform a normal
+	// deletion flow first, and if the normal deletion flow fails with a
+	// corrupt object error then it proceeds with the unsafe deletion
+	// of the object from the storage.
+	// NOTE: it skips precondition checks, finalizer constraints, and any
+	// post deletion hook defined in 'AfterDelete' of the registry.
+	// WARNING: This may break the cluster if the resource has
+	// dependencies. Use when the cluster is broken, and there is no
+	// other viable option to repair the cluster.
+	//
+	// TODO: it's located here for convenience so we can wire it
+	// from inside the handler.DeleteResource function.
```

Comment:
If it's a TODO, what will this eventually change to?

Reply:
I don't think it's necessary to change , `registry.Store` is used as a container for configuring the api handler from the storage, so the `Store` is a convenient holder for this, I removed the comment.